pdf_build_src/pandoc_script.py

```diff
@@ -17,19 +18,55 @@ def build_pdf(filename="bids-spec.pdf", logfile="bids-spec_pandoc_log.json"):
     logfile : str
         Name of the log file. Defaults to "bids-spec_pandoc_log.json".
     """
-    # Files that are not supposed to be built into the PDF
-    EXCLUDE = ["./index.md", "./schema/README.md", "./pregh-changes.md"]
-
-    # Get all input files
-    markdown_list = []
-    for root, dirs, files in os.walk("."):
-        for file in files:
-            fpath = os.path.join(root, file)
-            if fpath.endswith(".md") and fpath not in EXCLUDE:
-                markdown_list.append(fpath)
-            elif fpath.endswith("index.md"):
-                # Special role for index.md
-                index_page = fpath
+    # change directory name if running on CI
+    runs_on_CI = True if os.getenv("CIRCLE_BRANCH") else False
+    proj_fname = "bids-specification"
+    if runs_on_CI:
+        proj_fname = "project"
+
+    # location of this file: This is also the working directory when
+    # the pdf is being built using `cd build_pdf_src` and then
+    # `bash build_pdf.sh`
+    root = pathlib.Path(__file__).parent.absolute()
```

Comment:
I don't think we need to use `proj_fname`, so we don't need `runs_on_CI`. Also don't really like `root` to describe the containing directory of this file. Will make a suggestion below.

```suggestion
```